strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenArtifactControllerTest.java

```diff
@@ -1095,29 +1095,6 @@ private boolean checkSnapshotVersionExistsInMetadata(Metadata versionLevelMetada
                                    );
     }
 
-    /**
-     * User developer01 does not have general the ARTIFACTS_RESOLVE permission, but it's defined for single 'releases'
-     * repository. So because of dynamic privileges assignment they will be able to get access to artifacts in that
-     * repository.
-     */
-    @Test
-    @WithUserDetails("developer01")
-    public void testDynamicPrivilegeAssignmentForRepository()
```

Comment:
I think this test will need to be re-worked as well, instead of being dropped. You will have to create the user and carry out the operations as them. However, the `@WithUserDetails(...)` won't work that way, so you'll need to figure out how to do it differently.